### PR TITLE
Add monitoring to object cleaner

### PR DESCRIPTION
**Pull request by opudrovs**
Part of https://github.com/weaveworks/weave-gitops-enterprise/issues/3043

- Updated Explorer monitoring documentation to include the object cleaner metrics.

- Updated the Explorer dashboard screenshot.

Notes:
- Not sure if we need such a large screenshot which shows all parts of the Explorer dashboard. If not, I can restore the old screenshot.
- Naming suggestions are welcome.

**Questions:**
- Do we agree on calling the cleaner component officially everywhere Objects Cleaner or Object Cleaner?

I saw it called differently in different places. For now, I am calling it Objects Cleaner.

### CODE REVIEW

File: website/docs/explorer/operations.mdx [hunk 1]
```diff
@@ -225,6 +225,52 @@ indexer_inflight_requests{action="Add"} 0
 indexer_inflight_requests{action="Remove"} 0
 ```
 
+#### Management
+
+Explorer management path is composed of two components exporting metrics:
```
Comment (enekofb) [inline website/docs/explorer/operations.mdx:230]:
```suggestion
Explorer management contains the 'Objects Cleaner` component exporting metrics:
```

File: website/docs/explorer/operations.mdx [hunk 2]
```diff
@@ -225,6 +225,52 @@ indexer_inflight_requests{action="Add"} 0
...
+
+- Objects Cleaner Status
+- Objects Cleaner Remove Objects Requests
+
+The following metrics are available to monitor its health.
```
Comment (enekofb) [inline website/docs/explorer/operations.mdx:235]: this line might be better earlier as when you first introduce the metrics.

File: website/docs/explorer/operations.mdx [hunk 3]
```diff
@@ -225,6 +225,52 @@ indexer_inflight_requests{action="Add"} 0
...
+
+##### Objects Cleaner Status
+
+The metric `objects_cleaner_status` provides telemetry on the objects cleaner's `status` which can take on the following values:
+- Starting: objects cleaner is starting after starting the API server.
```
Comment (enekofb) [inline website/docs/explorer/operations.mdx:240]:
```suggestion
- Starting: Objects Cleaner is starting after starting the API server.
```

File: website/docs/explorer/operations.mdx [hunk 4]
```diff
@@ -225,6 +225,52 @@ indexer_inflight_requests{action="Add"} 0
...
+- Started: objects cleaner has been started and is watching for objects which are expired, according to their `RetentionPolicy`, to remove them from the datastore or indexer.
```
Comment (enekofb) [inline website/docs/explorer/operations.mdx:241]:
```suggestion
- Started: Objects Cleaner is watching for expired objects (according to their `RetentionPolicy`) to remove them from the stores.
```

File: website/docs/explorer/operations.mdx [hunk 5]
```diff
@@ -225,6 +225,52 @@ indexer_inflight_requests{action="Add"} 0
...
+- Stopped: objects cleaner has been stopped after stopping collection.
```
Comment (enekofb) [inline website/docs/explorer/operations.mdx:242]:
```suggestion
- Stopped: Objects Cleaner is stopped after stopping collection.
```

File: website/docs/explorer/imgs/explorer-query-metrics.png
Comment (enekofb) [inline website/docs/explorer/imgs/explorer-query-metrics.png:1]: for better UX we could have a picture with some more data points in the cleaners pannel

**Review by enekofb — APPROVED** [review]
added minor suggestions

File: website/docs/explorer/imgs/explorer-query-metrics.png
Comment (opudrovs) [reply website/docs/explorer/imgs/explorer-query-metrics.png:1]: Not a problem. But the original ticker interval is long, an hour or smth. like that. Is it OK if I add a very short ticker interval (locally, of course) to have some data points for the screenshot? For example, 30 seconds, which I used for testing.

**Comment by opudrovs** [comment]
@enekofb neat suggestions, thanks!

File: website/docs/explorer/operations.mdx (hunk 2 above)
Comment (opudrovs) [reply website/docs/explorer/operations.mdx]: Changed it to:

```
#### Management

Explorer management contains the 'Objects Cleaner` component exporting metrics. The following metrics are available to monitor its health:

- Objects Cleaner Status
- Objects Cleaner Remove Objects Requests

##### Objects Cleaner Status
```

File: website/docs/explorer/imgs/explorer-query-metrics.png
Comment (opudrovs) [reply website/docs/explorer/imgs/explorer-query-metrics.png:1]: I will update the screenshot after I am done with the enterprise PR changes and will run the app and accumulate data.

Comment (opudrovs) [reply website/docs/explorer/imgs/explorer-query-metrics.png:1]: Reverted to original screenshot for now. We'll update the screenshot after we can get for continuous data from staging.

**Pull request merged**